lib/crutches/format/number.ex

```diff
@@ -167,7 +167,8 @@ defmodule Crutches.Format.Number do
     number |> make_significant(precision)
   end
   defp prepare_as_rounded(number, precision, false) do
-    number = Float.round(number, precision)
+    multiplier = :math.pow(10, precision)
+    number = number |> Kernel.*(multiplier) |> Float.round(0) |> Kernel./(multiplier)
```

Comment:
I think the pipe operator makes this less readable. Can you change this line to:

```elixir
number = Float.round(multiplier * number, 0) / multiplier
```